functions.opengraph.php

```diff
@@ -100,10 +104,12 @@ function jetpack_og_tags() {
 
 		$tags['og:url']         = get_permalink( $data->ID );
 		if ( ! post_password_required() ) {
-			if ( ! empty( $data->post_excerpt ) ) {
-				$tags['og:description'] = preg_replace( '@https?://[\S]+@', '', strip_shortcodes( wp_kses( $data->post_excerpt, array() ) ) );
+			$excerpt = get_the_excerpt( $data );
```

Comment:
I'm afraid using `get_the_excerpt` (and the `the_content` filter below) would bring some of the problems we tried to fix in #2899. 

@kraftbj Could you chime in, since you worked on the original PR?